src/main/kotlin/hu/kirdev/schpincer/web/ConfigureController.kt

```diff
@@ -408,15 +408,18 @@ class ConfigureController {
     @Throws(FileNotFoundException::class, DocumentException::class)
     fun showOpenings(openingId: Long,
                      type: String?,
-                     request: HttpServletRequest): String {
+                     pageOrientation: String?,
+                     emptyRows: Int = 0,
+                     request: HttpServletRequest
+    ): String {
         val opening = openings.getOne(openingId)
         if (cannotEditCircle(opening.circle!!.id!!, request))
             return "redirect:/configure/" + opening.circle!!.id + "?error"
 
         val document = Document()
         val export = ExportType.valueOf(type!!)
 
-        if (export.isPortrait()) {
+        if (pageOrientation.equals("portrait")) {
```

Comment:
Don't use magic constants. They can cause you headache if you want to change or implement a new related feature and you need to look for all the possible values. Use `enum classes` if the set of possible values are predetermined and `const val` if they're no not. You can also avoid mistyping by that way.